syllabus/5-Substrate/3-Merklized-Storage_Slides.md

```diff
@@ -824,39 +827,84 @@ Meaning, if another client wants to sync polkadot, it should know the details of
 
 ## Additional Resources! 😋
 
-> Check speaker notes (click "s" 😉)
+- Check speaker notes (click "s" 😉).
+- Follows some additional content that is not covered.
 
 <img width="300px" rounded src="../../assets/img/5-Substrate/thats_all_folks.png" />
 
 Notes:
 
-- Shawn's deep dive: https://www.shawntabrizi.com/blog/substrate/substrate-storage-deep-dive/
+- Shawn's deep dive: <https://www.shawntabrizi.com/substrate/substrate-storage-deep-dive/>
```

Comment:
I'm not sure why you altered the links here. These are inside the notes and would show correctly. Adding the <> makes them actually wrong.
Also I suggest below the correct link - since this one does not exist anymore
```suggestion
- Shawn's deep dive: https://www.shawntabrizi.com/blog/substrate/substrate-storage-deep-dive/
```